docs/dev/reference/changelog.md

```diff
@@ -41,6 +41,81 @@ date: "2024-09-18"
 
 <!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->
 
+{{% changelog color="changed" title="Viam provisioning filename" date="2025-03-24" %}}
+
+When provisioning devices, the configuration file for provisioning is now called <FILE>viam-defaults.json</FILE> and uses different syntax and field names. The old <FILE>viam-provisioning.json</FILE> is still required for older versions of `viam-agent`.
```

Comment:
I'm not certain exactly how this should be phrased, but this isn't JUST a filename change. The new viam-defaults.json file allows setting ANY agent config values to customized defaults. So while it should now be used instead of the old, more limited viam-provisioning.json, it's not just a direct replacement.